src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -178,9 +231,10 @@ forward({
   to: walletCreatedFx,
 });
 forward({ from: multishardCreated, to: multishardCreatedFx });
+forward({ from: polkadotVaultCreated, to: polkadotVaultCreatedFx });
 
 sample({
-  clock: [walletCreatedFx.doneData, multishardCreatedFx.doneData],
+  clock: [walletCreatedFx.doneData, multishardCreatedFx.doneData, polkadotVaultCreatedFx.doneData],
```

Comment:
don't forget to add `polkadotVaultCreatedFx.doneData` for next sample (for select wallet after creation)